### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by usmanovmax**
Here is the russian translation for recording app. Should be properly done, everything is ready.

### CODE REVIEW

**Comment by usmanovmax** [comment]
Whoopsie, accidently wrote wrong stuff right there. Going to patch these right now. I apologize for all the weird stuff. Have never worked with Git and GitHub before

**Comment by usmanovmax** [comment]
Alright, everything should be fine now. Rechecked this one multiple times but because of an error in git decided to make a force-push. I want to believe everything will be fine for now...

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+#: src/MainWindow.vala:80
+msgid "Unable to Create an Audio File"
+msgstr "Невозможно Создать Аудиофайл"
+
+#: src/MainWindow.vala:81
+msgid ""
+"A GStreamer error happened while recording, the following error message may "
+"be helpful:"
+msgstr ""
+"Произошла ошибка "GStreamer" во время записи, следующее сообщение ошибки "
```
Comment (ryonakano) [inline po/ru.po]:
```suggestion
"Произошла ошибка \"GStreamer\" во время записи, следующее сообщение ошибки "
```

If you want to use quotation marks in your translations, you need to escape it with `\`.

**Comment by ryonakano** [comment]
Also, I would appreciate it if you would add Rusian translation in the `po/extra/` directory too.

**Comment by usmanovmax** [comment]
Supposingly, everything should be fine now. Nevertheless, thank you for your patience and calmness to all the mistakes I made

File: po/extra/ru.po [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+msgid "Audio Recording App"
+msgstr "Аудиозаписывающее приложение"
+
+#: data/com.github.ryonakano.reco.desktop.in:8
+msgid "com.github.ryonakano.reco"
+msgstr "com.github.ryonakano.reco"
+
+#: data/com.github.ryonakano.reco.desktop.in:11
+msgid "Record;Audio;Sound;Voice;"
+msgstr "Запись;Аудио;Звук;Голос"
```
Comment (ryonakano) [inline po/extra/ru.po]:
```suggestion
msgstr "Запись;Аудио;Звук;Голос;"
```

These strings are used to search this app from Application Menu, and each string should end with a semicolon (":").

**Comment by usmanovmax** [comment]
Alright, it's done. I rechecked other files mutliple times and it should be fine from now xD

**Pull request merged**